iceoryx_examples/icecrystal/Readme.md

```diff
@@ -2,6 +2,8 @@
 
 ## Introduction
 
+> This Readme.md is slightly outdated and not all functionality of the introspection is available with v1.0 
+
```

Comment:
Besides the introspection part I think that the expected output for the publisher and subscriber is wrong. What speaks against updating the readme?

Reply:
Is it ok for you to update this with the v1.0 release? I've removed out-commented the expected output.

Reply:
For me it is.